ClosedXML/Excel/PivotTables/XLPivotTable.cs

```diff
@@ -80,6 +81,122 @@ public IEnumerable<IXLPivotField> ImplementedFields
+                    .SetBaseField(v.BaseField)
+                    .SetBaseItem(v.BaseItem);
+
+                pivotValue.NumberFormat.NumberFormatId = v.NumberFormat.NumberFormatId;
+                pivotValue.NumberFormat.Format = v.NumberFormat.Format;
+            }
+
+            newPivotTable.Title = Title;
+            newPivotTable.Description = Description;
+            newPivotTable.ColumnHeaderCaption = ColumnHeaderCaption;
+            newPivotTable.RowHeaderCaption = RowHeaderCaption;
+            newPivotTable.MergeAndCenterWithLabels = MergeAndCenterWithLabels;
+            newPivotTable.RowLabelIndent = RowLabelIndent;
+            newPivotTable.FilterAreaOrder = FilterAreaOrder;
+            newPivotTable.FilterFieldsPageWrap = FilterFieldsPageWrap;
+            newPivotTable.ErrorValueReplacement = ErrorValueReplacement;
+            newPivotTable.EmptyCellReplacement = EmptyCellReplacement;
+            newPivotTable.AutofitColumns = AutofitColumns;
+            newPivotTable.PreserveCellFormatting = PreserveCellFormatting;
+            newPivotTable.ShowGrandTotalsColumns = ShowGrandTotalsColumns;
+            newPivotTable.ShowGrandTotalsRows = ShowGrandTotalsRows;
+            newPivotTable.FilteredItemsInSubtotals = FilteredItemsInSubtotals;
+            newPivotTable.AllowMultipleFilters = AllowMultipleFilters;
+            newPivotTable.UseCustomListsForSorting = UseCustomListsForSorting;
+            newPivotTable.ShowExpandCollapseButtons = ShowExpandCollapseButtons;
+            newPivotTable.ShowContextualTooltips = ShowContextualTooltips;
+            newPivotTable.ShowPropertiesInTooltips = ShowPropertiesInTooltips;
+            newPivotTable.DisplayCaptionsAndDropdowns = DisplayCaptionsAndDropdowns;
+            newPivotTable.ClassicPivotTableLayout = ClassicPivotTableLayout;
+            newPivotTable.ShowValuesRow = ShowValuesRow;
+            newPivotTable.ShowEmptyItemsOnColumns = ShowEmptyItemsOnColumns;
+            newPivotTable.ShowEmptyItemsOnRows = ShowEmptyItemsOnRows;
+            newPivotTable.DisplayItemLabels = DisplayItemLabels;
+            newPivotTable.SortFieldsAtoZ = SortFieldsAtoZ;
+            newPivotTable.PrintExpandCollapsedButtons = PrintExpandCollapsedButtons;
+            newPivotTable.RepeatRowLabels = RepeatRowLabels;
+            newPivotTable.PrintTitles = PrintTitles;
+            newPivotTable.SaveSourceData = SaveSourceData;
+            newPivotTable.EnableShowDetails = EnableShowDetails;
+            newPivotTable.RefreshDataOnOpen = RefreshDataOnOpen;
+            newPivotTable.ItemsToRetainPerField = ItemsToRetainPerField;
+            newPivotTable.EnableCellEditing = EnableCellEditing;
+            newPivotTable.ShowRowHeaders = ShowRowHeaders;
+            newPivotTable.ShowColumnHeaders = ShowColumnHeaders;
+            newPivotTable.ShowRowStripes = ShowRowStripes;
+            newPivotTable.ShowColumnStripes = ShowColumnStripes;
```

Comment:
Are there any reasons why assigning RelId and Theme is outstanding? (See line 109)

Reply:
`RelId` shouldn't be copied, because for a new pivot table, it will be autogenerated when saving. `Theme` should be implemented, yes.

Reply:
Done.

Reply:
Now we have `newPivotTable.Theme = Theme;` twice: line 111 and line 192.

Reply:
Fixed. I check for that, but obviously didn't go back far enough ;-)